praw/models/reddit/subreddit.py

```diff
@@ -18,6 +18,7 @@
 from .emoji import SubredditEmoji
 from .mixins import FullnameMixin, MessageableMixin
 from .modmail import ModmailConversation
+from .reasons import SubredditReasons
```

Comment:
While I appreciate the thought you put into making accessing this a bit simpler, I would prefer if this class is called `SubredditRemovalReasons`, and the file `removal_reasons`. It's possible, that Reddit adds something else called `reason` for a different purpose in the future, so I think it's better to be clear that these reasons are associated with item removal. Now that I think about it, isn't there a reason associated with a ban?

Reply:
While making the changes I also noticed that "reason" is used when updating the wiki. In any case, I made the changes w.r.t class/module names